library/common.h

```diff
@@ -181,7 +181,7 @@ inline void mbedtls_xor(unsigned char *r, const unsigned char *a, const unsigned
 #define MBEDTLS_ASM_AARCH64_PTR_CONSTRAINT "p"
 #elif UINTPTR_MAX == 0xfffffffffffffffful
 /* Normal case (64-bit pointers): use "r" as the constraint for pointer operands to asm */
-#define MBEDTLS_ASM_AARCH64_PTR_CONSTRAINT "r"
+#define MBEDTLS_ASM_AARCH64_PTR_CONSTRAINT "p"
```

Comment:
I tried that locally (as before, commenting out the inclusion of `string.h`) and both work:
```
aarch64-linux-gnu-gcc -I include -I library -Wall -Wextra -Werror -S library/constant_time.c
clang-15 -I include -I library -target aarch64 -Wall -Wextra -Werror -S library/constant_time.c
```

I don't know what the problem is in the [Travis build](https://app.travis-ci.com/github/Mbed-TLS/mbedtls/jobs/604652107).